src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const a of actions) {
```

Comment:
What is a?

Reply:
This is item in array) But I undestand that I incorrectly named the variables